### ISSUE TITLE
xNetBIOS: InterfaceAlias parameter no longer works

### ISSUE THREAD

**Issue by DelectableMechanisations** [PowerShell/xNetworking#322]
Hi Guys,

The most recent update to the xNetBIOS looks to have broken the InterfaceAlias parameter.

The Get/Set/Test-TargetResource functions now perform a CIM query filtering on the "Name' property.
````
$netAdapter = Get-CimInstance `
        -ClassName Win32_NetworkAdapter `
        -Filter ('Name="{0}"' -f $InterfaceAlias)
````

The previous (working) version of this resource used to query based on the "NetConnectionID" property:
````
$nic = Get-CimInstance `
        -ClassName Win32_NetworkAdapter `
        -Filter "NetConnectionID=`"$InterfaceAlias`""
````

On a Win32_NetworkAdapter CIM Instance the 'Name' property refers to the "InterfaceDescription" property returned by the `Get-NetAdapter` cmdlet.
````
PS C:\> Get-CimInstance Win32_NetworkAdapter | ft Name,NetConnectionId -auto

Name                                   NetConnectionId
----                                   ---------------
Microsoft Kernel Debug Network Adapter
vmxnet3 Ethernet Adapter               Ethernet0
Microsoft ISATAP Adapter
Microsoft Teredo Tunneling Adapter


PS C:\> Get-NetAdapter | ft -auto

Name      InterfaceDescription     ifIndex Status MacAddress        LinkSpeed
----      --------------------     ------- ------ ----------        ---------
Ethernet0 vmxnet3 Ethernet Adapter       5 Up     00-50-56-01-22-18   10 Gbps
````

To fix this resource this CIM query should filter on the "NetConnectionID" property.

Kind regards
David

**Comment by PlagueHO** [comment]
Hi @DelectableMechanisations - thank you for raising this issue. It was raised over here: https://github.com/PowerShell/xNetworking/issues/315

It has actually been resolved in this PR: https://github.com/PowerShell/xNetworking/pull/316

This will appear in the next release of xNetworking which is due to be released in a few weeks. You can use the dev branch if you wish. I'll close this issue for now and we can reopen it if the new fix doesn't resolve the problem.

Thanks again for raising this!

**Comment by matt6697** [comment]
Fix available in 5.7.0.0 (not 5.6.0.0 as stated in the changelog !)
